src/Compilers/CSharp/Portable/FlowAnalysis/EmptyStructTypeCache.cs

```diff
@@ -215,6 +215,7 @@ private FieldSymbol GetActualField(Symbol member, NamedTypeSymbol type)
                     return (!eventSymbol.HasAssociatedField || ShouldIgnoreStructField(eventSymbol, eventSymbol.Type)) ? null : eventSymbol.AssociatedField.AsMember(type);
                 case SymbolKind.Property:
                     // PROTOTYPE(semi-auto-props): Review other event associated field callers and see if we have to do anything special for properties.
+                    // Everything is reviewed except FlowAnalysis.
```

Comment:
Is this change still true after the revert?

Reply:
Yes. I reverted the fix for extra binding, but everything else is the same.